wasp/apps/flashlight.py

```diff
@@ -21,19 +20,31 @@ class TorchApp(object):
 
     def foreground(self):
         """Activate the application."""
-        self.draw()
         wasp.system.request_tick(1000)
+        wasp.system.request_event(wasp.EventMask.TOUCH)
 
         self._brightness = wasp.system.brightness
         wasp.system.brightness = 3
+        self.n_touch = 0
+        self.draw()
 
     def background(self):
-        """De-activate the application (without losing state)."""
+        """De-activate the application (without losing original state)."""
         wasp.system.brightness = self._brightness
 
     def tick(self, ticks):
         wasp.system.keep_awake()
 
     def draw(self):
         """Redraw the display from scratch."""
-        wasp.watch.drawable.fill(0xffff)
+        if self.n_touch % 6 < 3:
+            wasp.watch.drawable.fill(0xffff)
+        else:
+            wasp.watch.drawable.fill(0xf800)
+        print("ok")
+
+    def touch(self, event):
+        self.n_touch += 1
```

Comment:
_technically_ this will overflow at some point, even though in practice it would take far too many taps :)

```suggestion
        self.n_touch = (self.n_touch + 1) % 6
```